sherpa-onnx/csrc/offline-recognizer.h

```diff
@@ -56,7 +58,8 @@ class OfflineRecognizer {
   explicit OfflineRecognizer(const OfflineRecognizerConfig &config);
 
   /// Create a stream for decoding.
-  std::unique_ptr<OfflineStream> CreateStream() const;
+  std::unique_ptr<OfflineStream> CreateStream(
```

Comment:
I suggest that you create an overload version of CreateStream().
People who don't use context graph related stuff don't need to construct a vector. (You don't pay for what you don't need.)